docs/content/supported_tools/parsers/file/drheader.md

```diff
@@ -1,15 +1,185 @@
 ---
-title: "DrHeader"
+title: "DrHeader JSON Importer"
```

Comment:
If you change the title, you also have to change the name within the parser itself and also then, please create a db migrations file, otherwise unittests fail. So, I would suggest to leave the title as it is. 

Also, you can target this PR against bugfix to get it live on production faster.